src/scripts/split-into-songs.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# inspired by https://stackoverflow.com/questions/8544197/splitting-a-file-in-linux-based-on-content
+
+# create a helper to clean artist / song title
+friendly_name () { echo $1 | sed -e 's/\\//g' | sed -e 's/[^a-zA-Z0-9_.-]/-/g' | awk '{print tolower($0)}'; }
```

Comment:
& has been turned to - in "Florence & The machine" yielding "Florence---the-machine", we might want to change to "and" instead. I guess this comes from this line.

Reply:
Yes. If we want to improve this, maybe we could only replace the spaces with dashes.
```
friendly_name () { echo $1 | sed -e 's/\\//g' -e 's/\s/-/g' | awk '{print tolower($0)}'; }
```